src/main/resources/plugin.yml

```diff
@@ -297,10 +297,27 @@ permissions:
     griefprevention.lava:
         description: Grants permission to place lava near the surface and outside of claims.
         default: op
-    griefprevention.eavesdrop:
+    griefprevention.eavesdrop.*:
+        description: Allows a player to see signs, whispered chat messages (/tell) and softmuted messages.
+        default: op
+        children:
+            griefprevention.eavesdrop.messages.public: true
+            griefprevention.eavesdrop.messages.private: true
+            griefprevention.eavesdrop.signs: true
+            griefprevention.eavesdrop.immune: false
```

Comment:
wondering if this collides with eavesdrop.immune being set to `op` later on below.

Personally I'd remove asterisk permission groups, I'm not sure why we have it for `griefprevention.admin.` Speaking of which, that list would need to be updated anyways...

I guess for now, keep the admin.* for compatibility reasons (I imagine very few are using that), and just drop the groups for eavesdrop. I get wanting to make things easier for some groups of server admins, but IMO those who are using permission plugins with this plugin are few and it makes it easier to just have things individual without grouping...

Hmm, the more I look at this the more I want to modify it lol. Like instead of "public," it'd be "softmuted" or something idk.